docs/pages/versions/unversioned/sdk/bar-code-scanner.md

```diff
@@ -127,6 +127,24 @@ import { BarCodeScanner } from 'expo-barcode-scanner';
 
 ## Methods
 
+### `BarCodeScanner.requestPermissionsAsync()`
+
+Asks the user to grant permissions for accessing camera.
+
+On iOS this will require apps to specify both `NSCameraUsageDescription` and `NSMicrophoneUsageDescription` entries in the `Info.plist`
```

Comment:
as of #12860 only `NSCameraUsageDescription` is needed for expo-barcode-scanner: https://github.com/expo/expo/pull/12860/files#diff-fe45356ef2882ce2b4d565ed89bc8a48516318f6b1f52a9e2f14139b56184ed5R21-R22